ceph-ansible-pr-syntax-check/config/definitions/ceph-ansible-pr-syntax-check.yml

```diff
@@ -5,7 +5,7 @@
           url: https://github.com/ceph/ceph-ansible.git
           branches:
             - ${sha1}
-          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          refspec: +refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*
```

Comment:
looks like this PR has feature creep. Are you trying to introduce these enhancements aside from adding the Ceph Performance tests? or are they related?

Reply:
@alfredodeza thanks for reviewing this PR. a subset of this changeset was posted as #1392.